### ISSUE TITLE
Running project PluginWithCustomModule_Standalone plays painful white noise by default

### ISSUE THREAD

**Issue by rosshoyt** [eyalamirmusic/JUCECmakeRepoPrototype#15]
After cloning this repo and setting up a CLion project using the root CMakeLists.txt, I built and ran the _**PluginWithCustomModule_Standalone | Debug**_ configuration. Once running, the application blasted white noise out of my speakers. This were the audio output settings by default:

![default-audio-output](https://user-images.githubusercontent.com/43669429/185722002-3ff1d3ae-1acb-414b-a7ba-1b946ca42956.PNG)

My speakers were not on too loud at the time, but wanted to give you a heads up. I'd imagine other users might experience the same thing. I can look into a fix for this but don't have time until next week.

Regardless, thank you for these helpful CMake examples!

**Comment by jamierpond** [comment]
Sounds like a `juce::AudioBuffer` hasn't has been allocated without being cleared.

**Comment by eyalamirmusic** [comment]
Yes, this is intentional. That project just plays white noise from a code in a shared module.

If you feel this is too loud, however, I can certainly make it softer!

**Comment by rosshoyt** [comment]
@eyalamirmusic that might be a good idea! When I first ran the project, my speakers weren't on too loud yet it was still jarring when the sound started and I had to scramble to turn down speakers.
